INSTALL.md

```diff
@@ -22,25 +22,24 @@ Table of Contents:
 Please note: Mac users who want to use fink can skip down to Section 4.ii and
 use that to satisfy all dependencies before installing.
 
-i) Python (2.6 or 2.7 series), with some additional modules installed
----------------------------------------------------------------------
+i) Python (2.6, 2.7, or 3.4 series), with some additional modules installed
```

Comment:
I thought we were no longer supporting 2.6. https://github.com/GalSim-developers/GalSim/pull/703#discussion-diff-63630905L364